gotpm/cmd/seal.go

```diff
@@ -108,7 +108,18 @@ Thus, algorithm and PCR options are not needed for the unseal command.`,
 		defer srk.Close()
 
 		fmt.Fprintln(debugOutput(), "Unsealing data")
-		secret, err := srk.Unseal(&sealed, nil)
+
+		certifySel, err := getSelection()
+		if err != nil {
+			return err
+		}
+
+		var secret []byte
+		if len(certifySel.PCRs) > 0 {
+			secret, err = srk.Unseal(&sealed, tpm2tools.CertifyCurrent{PCRSelection: certifySel})
```

Comment:
This is fine as is, but I probably would have written this slightly differently (and I would have done the same in the prior PR):
```
cOpt := tpm2tools.CertifyCurrent{PCRSelection: certifySel}
if len(certifySel.PCRs) == 0 {
  cOpt = nil
}
secret, err = srk.Unseal(&sealed, cOpt)
```

Not a mandatory change, I just feel like it removes some duplication.

Reply:
done